From 09b6266a567b22e07200973312ffb8f43e7f7d43 Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Fri, 30 Apr 2010 11:12:24 +0200
Subject: [PATCH 01/45] Revert "omap3: beagle: Fix compile-time errors"

This commit clashes with the other beagle patches we apply on top

This reverts commit da5b291cba631d303cb137fa6a620c494d828197.
---
 arch/arm/mach-omap2/board-omap3beagle.c |   34 ++----------------------------
 1 files changed, 3 insertions(+), 31 deletions(-)

diff --git a/arch/arm/mach-omap2/board-omap3beagle.c b/arch/arm/mach-omap2/board-omap3beagle.c
index c5da58e..330fb25 100644
--- a/arch/arm/mach-omap2/board-omap3beagle.c
+++ b/arch/arm/mach-omap2/board-omap3beagle.c
@@ -56,22 +56,6 @@
 
 #define NAND_BLOCK_SIZE		SZ_128K
 
-#ifdef CONFIG_PM
-static struct omap_opp * _omap35x_mpu_rate_table	= omap35x_mpu_rate_table;
-static struct omap_opp * _omap37x_mpu_rate_table	= omap37x_mpu_rate_table;
-static struct omap_opp * _omap35x_dsp_rate_table	= omap35x_dsp_rate_table;
-static struct omap_opp * _omap37x_dsp_rate_table	= omap37x_dsp_rate_table;
-static struct omap_opp * _omap35x_l3_rate_table		= omap35x_l3_rate_table;
-static struct omap_opp * _omap37x_l3_rate_table		= omap37x_l3_rate_table;
-#else	/* CONFIG_PM */
-static struct omap_opp * _omap35x_mpu_rate_table	= NULL;
-static struct omap_opp * _omap37x_mpu_rate_table	= NULL;
-static struct omap_opp * _omap35x_dsp_rate_table	= NULL;
-static struct omap_opp * _omap37x_dsp_rate_table	= NULL;
-static struct omap_opp * _omap35x_l3_rate_table		= NULL;
-static struct omap_opp * _omap37x_l3_rate_table		= NULL;
-#endif	/* CONFIG_PM */
-
 static struct mtd_partition omap3beagle_nand_partitions[] = {
 	/* All the partition sizes are listed in terms of NAND block size */
 	{
@@ -377,21 +361,9 @@ static void __init omap3_beagle_init_irq(void)
 {
 	omap_board_config = omap3_beagle_config;
 	omap_board_config_size = ARRAY_SIZE(omap3_beagle_config);
-
-	if (cpu_is_omap3630()) {
-		omap2_init_common_hw(mt46h32m32lf6_sdrc_params,
-						mt46h32m32lf6_sdrc_params,
-						_omap37x_mpu_rate_table,
-						_omap37x_dsp_rate_table,
-						_omap37x_l3_rate_table);
-	} else {
-		omap2_init_common_hw(mt46h32m32lf6_sdrc_params,
-						mt46h32m32lf6_sdrc_params,
-						_omap35x_mpu_rate_table,
-						_omap35x_dsp_rate_table,
-						_omap35x_l3_rate_table);
-	}
-
+	omap2_init_common_hw(mt46h32m32lf6_sdrc_params,
+			     mt46h32m32lf6_sdrc_params, omap3_mpu_rate_table,
+			     omap3_dsp_rate_table, omap3_l3_rate_table);
 	omap_init_irq();
 #ifdef CONFIG_OMAP_32K_TIMER
 	omap2_gp_clockevent_set_gptimer(12);
-- 
1.6.6.1

